### PR TITLE
Faster interpolation using meta

### PR DESCRIPTION
**Pull request by Illviljan**
Removing a commented improvement that required that the minimum version requirement for dask to be increased.

- [x] Requires #5556
- [x] Passes `pre-commit run --all-files`

### CODE REVIEW

**Comment by Illviljan** [comment]
All the tests in test_missing.py was skipped because cftime was accidentally imported without it being installed.
* CI Additional / ubuntu-latest py37-min-all-deps (pull_request) - cftime installed -> failed test_missing.py
* CI Additional / ubuntu-latest py37-min-nep18 (pull_request) - cftime not installed -> pass because of a importorskip

**Comment by Illviljan** [comment]
Ah, there it is. Using meta only works from 2020.12 and onwards. I think we just change this to a version checker instead of increasing min version.

File: xarray/core/missing.py [hunk 1]
```diff
@@ -734,20 +734,34 @@ def interp_func(var, x, new_x, method, kwargs):
...
-            localize=localize,
-            concatenate=True,
-            dtype=dtype,
-            new_axes=new_axes,
-            # TODO: uncomment when min dask version is > 2.15
-            # meta=var._meta,
-            align_arrays=False,
-        )
+        if dask_version < "2020.12":
+            # Remove this whenever the minimum requirement for dask is 2020.12:
```
Comment (dcherian) [inline xarray/core/missing.py:738]: How about: 
```
if dask_version < "2020.12":
	meta = None
else:
	meta = var._meta

return da.blockwise(
	...,
	meta=meta
)
```

Comment (Illviljan) [reply xarray/core/missing.py:738]: Sure, I just thought that required more brainpower whenever someone cleans this up.

**Comment by dcherian** [comment]
Thanks @Illviljan

**Pull request merged**